README.md

```diff
@@ -1,36 +1,56 @@
 # github-security-jira
-GitHub Action for mapping security alerts to Jira tickets.
 
+GitHub Action for mapping security alerts to Jira tickets.
 
 ## Setup
 
 You need the following pieces set up to sync alerts with Jira:
 
 1. Two repo secrets containing a GitHub access token and a Jira API token, respectively.
-2. A workflow file which runs the action on a schedule, continually creating new tickets when necessary.
-
+1. A workflow file which runs the action on a schedule, continually
+   creating new tickets when necessary.
 
 ### Repo secrets
-The `reload/github-security-jira` action requires you to [create two encrypted secrets](https://help.github.com/en/actions/automating-your-workflow-with-github-actions/creating-and-using-encrypted-secrets#creating-encrypted-secrets) in the repo:
-
-1. A secret called `GitHubSecurityToken` which should contain a [Personal Access Token](https://help.github.com/en/github/authenticating-to-github/creating-a-personal-access-token-for-the-command-line) for the GitHub user under which this action should be executed. The token must include the `public_repo` scope if checking only public repos, or the `repo` scope for use on private repos. Also, the user must have [access to security alerts in the repo](https://help.github.com/en/github/managing-security-vulnerabilities/managing-alerts-for-vulnerable-dependencies-in-your-organization).
-2. A secret called `JiraApiToken` containing an [API Token](https://confluence.atlassian.com/cloud/api-tokens-938839638.html) for the Jira user that should be used to create tickets.
 
+The `reload/github-security-jira` action requires you to [create two
+encrypted
```

Comment:
Whats with all the odd linebreaks here?

Reply:
I went all in on what markdownlint complained about ... That's probably too much. I'll be a bit more gentle on it.